deepspeed/ops/transformer/inference/transformer_inference.py

```diff
@@ -710,6 +711,9 @@ def __init__(self,
         self.bias_residual_func = inference_cuda_module.bias_residual_fp16 if config.fp16 or config.q_int8 else \
                                     inference_cuda_module.bias_residual_fp32
 
+        self.residual_add_func = inference_cuda_module.residual_add_bias_fp16 if config.fp16 or config.q_int8 else \
```

Comment:
@RezaYazdaniAminabadi when running with `int8` quantization, the `residual_add_bias_fp16` needs to be called? Where we check the correct dtype of tensors?

Reply:
they should be fp16